x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/querydsl/container/QueryContainer.java

```diff
@@ -171,7 +171,7 @@ public QueryContainer(Query query,
 
             sortingColumns.add(new Tuple<>(Integer.valueOf(atIndex), comp));
         }
-        
+
```

Comment:
Again, unneeded formatting.

Reply:
Seems I have to disable the EditorConfig, otherwise IDEA is too clever the resets these settings.